src/messages.js

```diff
@@ -95,7 +95,7 @@ var warnings = {
   W018: "Confusing use of '{a}'.",
   W019: "Use the isNaN function to compare with NaN.",
   W020: "Read only.",
-  W021: "'{a}' is a function.",
+  W021: "'{a}' is a {b}.", // is a class or is a function
```

Comment:
I know you're just extending the warning, but while we're updating this message, why don't we make it something slightly more user-friendly. Saying "C is a class" doesn't really explain why it is a warning or how to fix it. What about

> Reassignment of '{a}', which is is a '{b}'. Use "var" or "let" to declare bindings that may change.